website/search/elastic_search.py

```diff
@@ -294,8 +296,13 @@ def update_node(node, index=None, bulk=False):
         else:
             es.index(index=index, doc_type=category, id=elastic_document_id, body=elastic_document, refresh=True)
 
+    #TODO @hmoco, make this a celry task that takes care of updating files
```

Comment:
Also, can this logic go in a blinker receiver in `website.file.listener` (similar suggestion in previous PR)?